### PR TITLE
Simplify commands

### PR DESCRIPTION
**Pull request by trowik**

### CODE REVIEW

File: app/mails/management/commands/sendmail.py [hunk 1]
```diff
@@ -39,61 +31,17 @@ def handle(self, *args, **kwargs):
...
-                    msg.attach(message.msg)
-                    msg.attach(add_text)
-
-            else:
-                msg = MIMEText(
-                    "\n\n".join((message.msg.get_payload(), str(text))),
-                    "plain",
-                    charset,
-                )
+            send_email_with_attachments(message=message, mail=mail, text=text, msg=msg)
```
Comment (winged) [inline app/mails/management/commands/sendmail.py:39]: Sure you're calling the right function here?

Also, `msg` is always `None`, why even pass it?

File: app/mails/management/commands/sendmail.py [hunk 2]
```diff
@@ -108,4 +56,53 @@ def handle(self, *args, **kwargs):
                 message.delete()
                 mail.delete()
 
-        smtp.quit()
+
+def attach_MIMEText_to_mulitpart_messages(message, mail, text, msg):
```
Comment (winged) [inline app/mails/management/commands/sendmail.py:60]: This function is never called (see also above)

Comment (winged) [reply app/mails/management/commands/sendmail.py:60]: Also, the `msg` parameter is not really used. You can leave it out

**Review by winged — CHANGES_REQUESTED** [review]
Looks better, but still has some problems to fix.

Also, no need to delete and re-create PRs, you can push new code to the same PR. Helps to keep review history

**Comment by trowik** [comment]
That's what I normally do, but for unknown reasons the commit history got messed up and I didn't have the energy to clean it up so I just opened a new PR.

File: app/mails/management/commands/sendmail.py (hunk 1 above)
Comment (trowik) [reply app/mails/management/commands/sendmail.py]: wrong function, changed it to the correct one

Comment (trowik) [reply app/mails/management/commands/sendmail.py]: also forgot to make `msg` global, it's used in another function as well

**Review by winged — APPROVED** [review]
Nice work! :tada:

**Pull request merged**